tests/test_servicex.py

```diff
@@ -797,6 +857,54 @@ async def test_stream_parquet_files_from_minio(mocker):
 
     assert mock_servicex_adaptor.query_json["result-format"] == "parquet"
 
+@pytest.mark.asyncio
```

Comment:
Have you run all your modified files through black? I'm seeing some differences in the formatting here which is making me think that didn't happen.